src/utils/getItems.ts

```diff
@@ -0,0 +1,9 @@
+export function getActiveItems<T extends { completed: boolean }>(items: T[]) {
+  return items.filter(item => !item.completed).length;
+}
```

Comment:
```suggestion
export function getActiveItems(items: Todo[]): number {
  return items.filter(item => !item.completed).length;
}
```
looks complicated
you just need count so